test.tf

```diff
@@ -108,24 +123,99 @@ resource "azurerm_postgresql_server" "example" {
   sku_name                     = "B_Gen5_2"
```

Comment:
**Accurics remediation**
 
**[LOW]: Ensure that 'Auditing' Retention is 'greater than 90 days' for Azure SQL Server**
 
**Vulnerability**
'Auditing' Retention is period is less than 90 days' for Azure SQL Server. This can impact the availability of sql server data
 
**Remediation**
Auditing an instance of the SQL Server Database Engine or an individual database involves tracking and logging events that occur on the Database Engine. SQL Server audit lets you create server audits, which can contain server audit specifications for server level events, and database audit specifications for database level events. Audited events can be written to the event logs or to audit files.

source: accurics-beta